skytap/resource_skytap_environment.go

```diff
@@ -116,7 +119,7 @@ func resourceSkytapEnvironmentCreate(d *schema.ResourceData, meta interface{}) e
 	}
 
 	log.Printf("[INFO] environment create")
-	log.Printf("[DEBUG] environment create options: %#v", spew.Sdump(opts))
+	log.Printf("[DEBUG] environment create options: %v", spew.Sdump(opts))
```

Comment:
TRACE?

Reply:
don't mind if TRACE or DEBUG but probably `Sdump` provides too much information for a `DEBUG `level in a normal case.